controller/persistence/api_session_store.go

```diff
@@ -46,14 +46,14 @@ const (
 
 type ApiSession struct {
 	boltz.BaseExtEntity
-	IdentityId      string
-	Token           string
-	IPAddress       string
-	ConfigTypes     []string
-	MfaComplete     bool
-	MfaRequired     bool
-	LastActivityAt  time.Time
-	AuthenticatorId string
+	IdentityId      string    `json:"identity_id"`
```

Comment:
So we are adding json tagging to everything? Also, was there reasoning for snasnake ke vs camel case?